kymatio/scattering2d/__init__.py

```diff
@@ -1,4 +1 @@
-__all__ = ['Scattering2D']
-
-
-from .scattering2d import Scattering2D
+from .frontend import *
```

Comment:
Is it necessary to import `*` or can we be explicit? It should be safe, but if it is avoidable, it might as well be avoided. I also don't yet see what this file contains

Reply:
i agree